src/attesters/sev-snp/collect_evidence.c

```diff
@@ -0,0 +1,105 @@
+/* Copyright (c) 2022 Intel Corporation
+ * Copyright (c) 2020-2022 Alibaba Cloud
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "sev_snp.h"
+#include <fcntl.h>
+#include <linux/sev-guest.h>
+#include <rats-tls/attester.h>
+#include <rats-tls/log.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/ioctl.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
```

Comment:
Usually, the headers should be arranged in this order:
```
// standard headers
#include <string.h>
// 3rd
#include <openssl/sha.h>
#include <openssl/ts.h>
// rats-tls specific
#include <rats-tls/log.h>
// local headers
#include "crypto.h"
```

Please follow this rule to arrange the order of headers.